netwerk/necko.properties

```diff
@@ -80,6 +80,12 @@ CookieRejectedInvalidPrefix = Cookie “%1$S” has been rejected for invalid pr
 CookieRejectedInvalidPath = Cookie “%1$S” has been rejected for invalid path.
 # LOCALIZATION NOTE (CookieRejectedInvalidCharValue): %1$S is the cookie name.
 CookieRejectedInvalidCharValue = Cookie “%1$S” has been rejected for invalid characters in the value.
+# LOCALIZATION NOTE (CookieRejectedAttributePathOversize): %1$S is the cookie name.
+CookieRejectedAttributePathOversize = Cookie “%1$S” has been rejected because its path attribute is too big.
+# LOCALIZATION NOTE (CookieRejectedAttributeDomainOversize): %1$S is the cookie name.
+CookieRejectedAttributeDomainOversize = Cookie “%1$S” has been rejected because its path attribute is too big.
```

Comment:
I think this is wrong. Pinged baku, we could fix it without a new ID if we're quick (can't access the bug or phabricator diff).